DOC Tidy up writing style of performance section (#766)
silverstripe/developer-docs: en/02_Developer_Guides/08_Performance/06_ORM.md

# ORM performance

## Identifying ORM performance bottlenecks

ORM performance issues can arise from various factors, including:

- Inefficient database queries
- Lack of proper indexes
- Large datasets
- Complex relationships

> You can use [lekoala/silverstripe-debugbar](https://github.com/lekoala/silverstripe-debugbar?tab=readme-ov-file#database-profiling) to identify slow running database queries.

Reviewer: That's not CMS 6 compatible currently.
Maybe call out the [`showqueries` URL variable](https://docs.silverstripe.org/en/6/developer_guides/debugging/url_variable_tools/#database) instead.

Author: Updated

> You can append [?showqueries=1](https://docs.silverstripe.org/en/6/developer_guides/debugging/url_variable_tools/#database) to the URL on a dev environment to identify slow running database queries.

Reviewer: ```suggestion
You can use the [`showqueries` variable tool](/developer_guides/debugging/url_variable_tools/#database) on a dev environment to identify slow running database queries.
```
`showqueries` accepts a few different values that give different output, e.g.`showqueries=inline` will show you the values in the SQL instead of `?` parameter placeholders.
This is also consistent with one other link to that page that I'm aware of

Author: Updated

## Indexes

> You can define indexes for your ORM queries using the `$indexes` configuration property in your `DataObject` subclasses. See the [Indexes](/developer_guides/model/indexes) section for more information. Adding indexes to frequently queried fields can significantly improve performance.

Reviewer: ```suggestion
Adding indexes to frequently queried fields can significantly improve performance. You can define indexes for your ORM queries using the `$indexes` configuration property in your `DataObject` subclasses. See the [Indexes](/developer_guides/model/indexes) section for more information.
```

Author: Updated

```

> [!WARNING]

> > Excluding models from database checks can lead to undetected data corruption or other issues. Only exclude models if you are certain of what you are doing. Consider testing these changes thoroughly in a development environment before applying them to production.

Reviewer: ```suggestion
> Excluding models from database checks can lead to undetected data corruption or other issues. Only exclude models if you are certain of what you are doing.
```
The problem with it is more of a "down the line the checks might fail" - checking it works fine once is likely to be a footgun as you may be fooled into thinking "it worked this time so it'll always work"

Author: Updated
